apps/overige-objecten-api/src/utils/getVacData copy.ts

```diff
@@ -0,0 +1,52 @@
+import kebabCase from 'lodash.kebabcase';
```

Comment:
Klopt het dat de filename `getVacData copy.ts` is? Met een spatie?

Reply:
Deze is per ongeluk gekopieerd en moet worden verwijderd.